microhapulator/workflows/analysis.smk

```diff
@@ -269,7 +269,9 @@ rule plot_haplotype_calls:
     run:
         result = TypingResult(fromfile=input.result)
         mhapi.plot_haplotype_calls(
-            result, f"analysis/{wildcards.sample}/03typing/callplots", sample=wildcards.sample
+            result,
+            "analysis/{}/03typing/callplots".format(wildcards.sample),
+            sample=wildcards.sample,
```

Comment:
Snakemake has weird problems with f-strings in Python 3.12, so calling the `str.format` function in a few places.